src/lm/const-arpa-lm.cc

```diff
@@ -278,7 +278,9 @@ void ConstArpaLmBuilder::ConsumeNGram(const NGram &ngram) {
                            cur_order == ngram_order_ - 1,
                            ngram.logprob, ngram.backoff);
 
-    KALDI_ASSERT(seq_to_state_.find(ngram.words) == seq_to_state_.end());
+    if (seq_to_state_.find(ngram.words) != seq_to_state_.end()) {
+      KALDI_ERR << "ngram " << ngram.words << " appears twice in the arpa file";
```

Comment:
I see where that template is defined now, it's in kaldi-error.h.
But I'd like it removed, it could easily cause confusing bugs and bad interactions with other libraries.  Can you remove it in this PR also? I tried compiling without it and it doesn't seem to be used anywhere.  But if you find test programs that use it, change them.